skyblock docs (#3)
TrixNetwork/docs: SkyBlock/README.md

# SkyBlock

![](Skyblock_Island.webp "SkyBlock Island")

## Introduction

> **Skyblock** is one of the most popular survival gamemodes. The idea is that you can create and expand your very own world infinitely with only minimal materials. You start on a small island floating in the middle of void with a few items and a single tree and you must learn to use and preserve your materials wisely as each one has a strategic use, (for example lava buckets), and to know the basics of Minecraft in order to complete challenges and to expand. It is normal to make beginner mistakes, sometimes requiring many restarts.

Reviewer: ambiguous text.

```suggestion
**Skyblock** is one of the most popular survival gamemodes. The gamemode is about creating and expanding your very own world infinitely with only minimal materials. You start on a small island floating in the middle of void with a few basic items and a single tree. You  must survive and expand this island. You must learn to use and preserve your materials wisely, since each material has a strategic use, and wasting any of them could be disastrous. For example, lava buckets have a *very* important use! You need to also know the basics of Minecraft in order to complete challenges and expand. 

It is normal to make mistakes, that may even require restarts, when you just start playing.
```

## Commands

On TrixMC we have a set of commands that you can use to manage your SkyBlock world. The commands are of the format `/is <command>`. The commands are listed below:

> ### controlpanel

Reviewer 2: ```suggestion
### `controlpanel`
```
make all the titles also codeblocks, looks nicer imo

Author: Done

### `create`

`/is create` is used to create your island.

### `fly`

`/is fly` is used to toggle flying on your island. More info about fly can be found in [Fly](#fly-1).

### `generator`

`/is generator` is used to buy and upgrade your cobblestone generator. More info about generator can be found in [Generator](#generator-1).

### `go`

`/is go` is used to teleport to your island.

### `info`

`/is info` is used to view island information.

### `language`

`/is language` is used to change the language.

### `level`

`/is level <player>` is used to calculate your island level or mentioned player's level.

### `reset`

`/is reset` is used to reset your island. WARNING: ALL PROGRESS WILL BE LOST IF YOU USE THIS COMMAND.

### `sethome`

`/is sethome` is used to set your home teleport point.

### `setname`

`/is setname` is used to set the name of your island.

### `settings`

`/is settings` is used to view and change your island settings. More info about settings can be found in [Settings](#settings-1).

### `resetname`

`/is resetname` is used to reset the name of your island.

### `unban`

`/is unban <player>` is used to unban a player from your Island.

### `team`

> 1. `/is team` is used to manage your team.

Reviewer: I think a bulleted list would make more sense here.

Author: This way commands are on the sidebar and it helps in easy navigation.

### `warp`

`/is warp <player>` is used to warp to a player's island.

### `warps`

`/is warps` is used to list all available warps on TrixMC SkyBlock. More info on warps can be found in [Warps](#Warps-1).

### `bank`

`/is bank deposit <amount>` is used to deposit money into your island bank from your wallet.
`/is bank withdraw <amount>` is used to withdraw money into your island bank from your wallet.
`/is bank balance` is used to view your island bank balance.
`/is bank statement` is used to view your island bank statement.

More info on money and bank can be found in [Money](#Money).

### `balance`

`/balance` is used to check your wallet balance.

More info on money and bank can be found in [Money](#Money).

### `chat`

`/is chat` is used to toggle island chat on and off. This allows you to communicate with ONLY your teammates.

### `teamchat`

`/is teamchat` is used to toggle team chat. This allows you to communicate with ONLY those in your team.

## Island Types

There are many different types of islands you can choose from while creating an island on TrixMC SkyBlock.

### List of island Types

A list of island types shall be added in the near future.

## Money

> As you progress, you will need to use money and make transactions. Money is stored in one of two places: wallet or bank. You can see the balance of both on the right hand side of your screen.

Reviewer: ```suggestion
As you progress, you will need to use money and make transactions. Money is stored in one of two places: the **wallet** or the **bank**. You can see the balance in both on the right hand side of your screen.
```

### Interest

> For money stored in your bank, you recieve an interest. The interest is credited every hour you are online. The rate of interest is not revealed and keeps changing.

Reviewer: ```suggestion
For money stored in your bank, you recieve an interest. The interest is credited for every hour you are online. The rate of interest is not revealed and keeps changing.
```

## Challenges

> Challenges let you complete various tasks and recieve rewards. These rewards are in the form of items and experience. These items are useful in your journey to beat the gamemode. You can check your challenges using the command [`/is challenges`](#challenges).

Reviewer: But what is beating the gamemode? It isn't clear

Author: ```suggestion
Challenges let you complete various tasks and receive rewards. These rewards are in the form of items and experience. These items are useful in your journey to make progress in the gamemode. You can check your challenges using the command [`/is challenges`](#challenges).
```

### List of Challenges

This list shall be added in the near future.

## Settings

Settings allow you to customize your island. You can view and change your settings using the command [`/is settings`](#settings).

There are currently 59 Protection Settings, 6 General Settings and 17 SkyBlock Island Settings that can be applied to the SkyBlock Island. Each of which ENABLE or DISABLE access to one or more types of players: Visitor, Coop, Trusted, Member, Sub-Owner and Owners for that island.

### List of settings

> <!-- The below table was copy pasted from the BentoBox wiki -->

Reviewer: Don't use HTML in markdown. please.

Author: HTML tables are easier and more intuitive than md.

> <table width="100%" align="center"> <tbody><tr> <td align="left" valign="top"><b>Settings</b></td> <td align="left"><b>Flags to enable or disable</b></td> </tr> <tr> <td align="left" valign="top">Protection Settings</td> <td align="left">Animal riding, Anvils, Armor stands, Beacons, Beds, Break blocks, Breed animals, Brewing stands, Buckets, Buttons, Chorus fruits, Collect lava, Collect water, Egg throwing, Enchanting Table, End Portal, EnderPearls, Experience pickup, Fire, Fire extinguish, Fish scooping, Frost walker, Furnace, Gates, Hurt animals, Hurt monsters, Hurt villagers, Item drop, Item pickup, Jukebox use, Leash use, Lever use, Lock island, Milking, Mount inventory, Name tags, Nether Portal, Note block, Place blocks, Pressure Plates, Redstone items, Shearing, Spawn eggs, Next Page (sign), TNT damage, Trample crops, Trap doors, Turtle Eggs, Use doors, Villager trading, Workbenches, Use containers, Use dispensers, Use droppers, Use hoppers, Use Item Frames, Throw Potions, Previous Page (sign)</td> </tr> <tr> <td align="left" valign="top">General Settings</td> <td align="left">Animal spawning, End PVP, Fire spread, Monster spawning, Nether PVP, Overworld PVP</td> </tr> <tr> <td align="left" valign="top">BSkyBlock Settings</td> <td align="left">Chest damage, Clean Super Flat, Coarse dirt tilling, Creeper damage, Creeper griefing, Ender Chests, Enderman griefing, Enter/Exit messages, Island respawn, Offline Redstone, Piston Push, Remove monsters, Obsidian Scooping, Feather Fall Teleporting, Command Ranks, Invincible Visitors, Limit mobs to island</td> </tr> </tbody></table>

Reviewer: Don't use HTML in markdown. please.

Author: HTML tables are easier and more intuitive than md.

Reviewer: 😭😭😭
